src/Elkuent/Builder.php

```diff
@@ -207,6 +215,56 @@ public function getFresh($columns = array())
     }
 
     /**
+     * Specify how to aggregate results
+     * Structured as an Elasticsearch aggregation: https://www.elastic.co/guide/en/elasticsearch/reference/1.5/search-aggregations.html
+     * 
+     * @param  array $aggregation PHP array representation of JSON aggregation object
+     * @return void
+     */
+    public function aggregateBy($aggregation) 
```

Comment:
On second thought, I guess the "raw" output is not json either, but a PHP array. :) To mirror the output format, maybe we don't need to support json input for this method, after all.